hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/test/TestProcessorSupplierContext.java

```diff
@@ -71,9 +75,24 @@ public int memberIndex() {
         return memberIndex;
     }
 
-    /**
-     * Set the member index.
-     */
+    @Nonnull @Override
+    public File attachedDirectory(@Nonnull String id) {
+        return attachedDirectory;
```

Comment:
We should store the files in a map, keyed by the `id`. This might be enough fur current cases, but it's an incorrect implementation.